.springBeans

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally such files should go to .gitignore so that it does not appear in the repo. Such files are local to you.